--- org.argeo.tp.build/biz.aQute.bndlib/src/aQute/libg/re/Catalog.java.orig	2025-04-06 09:37:31.293814627 +0200
+++ org.argeo.tp.build/biz.aQute.bndlib/src/aQute/libg/re/Catalog.java	2025-04-06 09:38:14.810683880 +0200
@@ -26,7 +26,7 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
-import org.eclipse.jdt.annotation.Nullable;
+//import org.eclipse.jdt.annotation.Nullable;
 
 import aQute.libg.re.RE.C;
 import aQute.libg.re.RE.F;
@@ -193,7 +193,7 @@
 	 * @param res the members
 	 * @return a group, either named or capturing
 	 */
-	public static RE term(@Nullable
+	public static RE term(
 	String name, RE... res) {
 		assert isValidGroupName(name) : name;
 		if (res == null || res.length == 0)
